.gitignore

```diff
@@ -59,3 +59,9 @@ typings/
 
 # next.js build output
 .next
+
+# TypeScript build output
+build/*
+
+# Development environments
+.vscode
```

Comment:
This icon means you're missing a trailing newline.

Trailing newlines help to make the next diff a little more readable, because the lines are only "add" and not "change".

Reply:
Done https://github.com/knative-portability/Kubercade/pull/1/commits/44ebadc6b2b505e83f4f3a5ee84d1551a769f4fd